ReflectometryServer/parameters.py

```diff
@@ -104,6 +108,16 @@ def rbv(self):
         """
         return self._rbv()
 
+    @property
+    def rbv_at_sp(self):
+        """
+        Returns: Does the read back value match the set point target within a defined tolerance
+        """
+        if self.rbv is None or self._set_point_rbv is None or abs(self.rbv - self._set_point_rbv) > self.rbv_to_sp_tolerance:
```

Comment:
Add and is in beam so that a displacement which is out of the beam does not get marked as incorrect

Reply:
I created overriding `rbv_at_sp` functions in the `TrackingPosition` and `InBeamParameter` child classes to include their component specific `in_beam` checks.